### DISCUSSION TITLE
Welcome to search-index Discussions!

### DISCUSSION THREAD

**Discussion by fergiemcdowall** [fergiemcdowall/search-index#589] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by eklem** [comment]
Nice, @fergiemcdowall ❤️ 
I'll be sure to take all my search ideas and questions here 😄

**Reply by fergiemcdowall** [reply]
please do 👍🙂
